Tests/FunctionalTests/NetworkResponseHandlerFunctionalTests.swift

```diff
@@ -318,6 +319,71 @@ class NetworkResponseHandlerFunctionalTests: FunctionalTestBase {
         XCTAssertTrue(storeResponsePayloadManager.getActivePayloadList().isEmpty)
     }
 
+    /// Tests that when an event is processed after a persisted reset event that the store payloads are saved
+    func testProcessResponseOnSuccess_afterPersistedResetEvent_savesStorePayloads() {
+        dataStore.set(key: EdgeConstants.DataStoreKeys.RESET_IDENTITIES_DATE, value: Date().timeIntervalSince1970) // date is before `event.timestamp`
+        networkResponseHandler = NetworkResponseHandler() // loads reset time on init
+
+        let event = Event(name: "test", type: "test-type", source: "test-source", data: nil)
```

Comment:
If the test executes fast enough, and since TimeInterval is in seconds, could the RESET_IDENITTIES_DATE and Event timestamp be the same?  Could you instead set the stored RESET_IDENTITITIES_DATE to a time in the past?